src/shared/components/community/sidebar.tsx

```diff
@@ -144,10 +144,10 @@ export class Sidebar extends Component<SidebarProps, SidebarState> {
             {myUSerInfo && this.blockCommunity()}
             {!myUSerInfo && (
               <div className="alert alert-info" role="alert">
-                {i18n.t("community_not_logged_in_alert", {
-                  community: name,
-                  instance: hostname(actor_id),
-                })}
+                {i18n.t("community_not_logged_in_alert")}
```

Comment:
Try using the T component. Here's a code snippet example from the modlog component:

```
<T i18nKey="modlog_content_warning" class="d-inline">
    #<strong>#</strong>#
</T>
```